event-algo.bs

```diff
@@ -2073,10 +2183,16 @@ Issue: Should these be moved into the Clipboard spec?
     1. Let |data| = the text on the clipboard being pasted
     1. Let |result| = <a>fire an InputEvent</a> with "beforeinput", "insertFromPaste" and |data|
```

Comment:
How about `insertFromPasteAsQuotation`? It's currently supported only by Gecko.

Reply:
I think that's a separate issue, filed https://github.com/w3c/uievents/issues/368